benchalerts/tests/integration_tests/test_alert_pipeline.py

```diff
@@ -32,7 +32,9 @@ def test_alert_pipeline(monkeypatch: pytest.MonkeyPatch, github_auth: str):
     # note: something *might* go wrong if we go past 1000 statuses on this test commit?
     # https://docs.github.com/en/rest/commits/statuses#create-a-commit-status
     test_status_repo = "conbench/benchalerts"
-    test_status_commit = "4b9543876e8c1cee54c56980c3b2363aad71a8d4"
+    # Note: this will start failing after N test executions, see
+    # https://github.com/conbench/conbench/issues/945
```

Comment:
Thanks for adding to the comment :) I'm relieved to see that I anticipated this problem a while ago on line 33!

Reply:
@austin3dickey to be honest, I did this patch sooo quickly that I didn't even see your comment :) :racehorse: